cmake/python.cmake

```diff
@@ -1,5 +1,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
+# On Windows, instruct Python to output UTF-8 even when not
+# interacting with a terminal. This is required since Python scripts
+# are invoked by CMake code.
```

Comment:
Just for the record  and sake of the reader, can you add a comment saying something like this? "On Windows, standard I/O encoding defaults to the current code page if not connected to a terminal, which is often not what we want."

I'm fine either way, but thought this would be useful.

Reply:
Good suggestion, done.